src/clj/kixi/hecuba/api/devices.clj

```diff
@@ -118,17 +118,22 @@
 
 (defmethod calculated-sensor :default [sensor])
 
+(defn create-synth-sensors [sensors]
+  (flatten (->> sensors
+                (mapcat (fn [sensor]
+                          (if (= "CUMULATIVE" (:period sensor))
+                            [sensor (ext-type sensor "differenceSeries")] [sensor])))
+                (mapcat (fn [sensor]
+                          (if (= "PULSE" (:period sensor))
+                            [sensor (calculated-sensor sensor)] [sensor]))))))
```

Comment:
I don't think this will work as M^3 and FT^3 return vectors of sensors rather than specific sensors. I think we should change the KWH calculated-sensor to return a vector as well and then change FT^3, M^3 and create-synth-sensors to deal with a vector of sensors coming back